book.tex

```diff
@@ -1018,7 +1018,9 @@ \subsection{Syntax}
 IN$: 
 \end{ffcode}
 
-\subsection{eq Attribute}
+\subsection{float Attribute}
```

Comment:
@c71n93 Can we make it plural? I believe we will mention other `float` attributes in the next updates of this book

Reply:
Yes, it should be plural, that's typo. Thanks.